ntbs-build/openshift-pipelines/pipelines/ntbs-update-live-routes.yml

```diff
@@ -0,0 +1,77 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: ntbs-update-live-routes
+  namespace: ntbs-build
+spec:
+  params:
+    - default: live
+      description: The branch from which to take the deployment file
+      name: branch
+      type: string
+    - default: live
+      description: Which service to point live routes at (coming-soon or ntbs)
```

Comment:
I wonder if we might be better having two separate pipelines - one for entering the maintenance window and one for coming out of it.

Pros:

- Easier to use (users wouldn't need to set any variables, as they'd almost always be using the live branch which is the default)
- Wouldn't need the check-input step

Cons:

- Duplication of code - less maintainable

I think maybe the ease of use is the one that's most important to us. What are your thoughts?

Reply:
I have made 2 pipelines